src/System.CommandLine/Parsing/ParseResult.cs

```diff
@@ -64,12 +64,20 @@ public class ParseResult
 
         public IReadOnlyCollection<string> UnparsedTokens { get; }
 
+        // FIX: (ParseResult) remove
         public object? ValueForOption(string alias) =>
             ValueForOption<object?>(alias);
         
+        public object? ValueForOption(Option option) =>
+            ValueForOption<object?>(option);
+
+        // FIX: (ParseResult) remove
```

Comment:
I am not sure what this comment means. Is there more to do here?

Reply:
Just a note to self. Removed.